build/NuSpecs/WindowsAppSDK-Nuget-Native.targets

```diff
@@ -30,4 +30,11 @@
     </Reference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.ApplicationModel.Resources.winmd">
+      <Private>false</Private>
+      <Implementation>Microsoft.ApplicationModel.Resources.dll</Implementation>
+    </Reference>
+  </ItemGroup>
```

Comment:
if nuget doesn't handle this automatically, i am afraid WinUI may need some change to ingest.

Reply:
good point - will check that